Pod/Classes/RZMessagingWindow.m

```diff
@@ -288,4 +289,84 @@ - (void)configurePassThroughView
 #pragma mark - RZRootMessagingViewController
 
 @implementation RZRootMessagingViewController
+
+#pragma mark - Orientation method overrides
+
+-(BOOL)shouldAutorotate
+{
+    UIViewController *topViewController = [RZRootMessagingViewController topViewController];
+    
+    if ( topViewController == self ) {
+        // If the top view controller is itself then do the same as
+        // default by returning YES.
+        return YES;
+    }
+    else {
+        return topViewController.shouldAutorotate;
+    }
+}
+
+- (NSUInteger)supportedInterfaceOrientations
+{
+    UIViewController *topViewController = [RZRootMessagingViewController topViewController];
+    
+    if ( topViewController == self ) {
+        // If the top view controller is itself then do the same as
+        // default by:
+        // 1)For iPhone, all except upside down
```

Comment:
Standard practice is to skip upside down on iPhone, so that if you get a phone call, you don't hold the phone up to your ear upside down. Recommended by Apple, but I don't remember where and couldn't find it when I searched.